app/templates/explore.hbs

```diff
@@ -5,7 +5,7 @@
   <div class="twelve wide column">
     <h1 class="ui header">{{t 'Events'}}</h1>
     {{#each filteredEvents as |event|}}
-      {{event-card event=event isWide=true shareEvent=(action 'shareEvent')}}
+      {{event-card event=event isWide=true category=category sub_category=sub_category event_type=event_type shareEvent=(action 'shareEvent')}}
```

Comment:
Use `camelCase` for the attributes on the `event-card` 

```hbs
subCategory=sub_category eventType=event_type
```

Reply:
Actually I was trying to keep the format used consistent. Because in the same file this format is used. Should I change the attributes in both lines to camel case?

Reply:
We use `camelCase` for all js parameters/variables/function with the exception of query parameters. We use `snake_case` for query parameters. 

So, it will be `sub_category` inside `explore` to denote it is binded to a query parameter and should be `subCategory` within the `event-card` component.

Reply:
Ok. I'll make the necessary changes to the parameters in both the lines